src/Blog.res

```diff
@@ -39,44 +39,34 @@ module Badge = {
     </div>
   }
 }
-module CategorySelector = {
-  type selection =
-    | All
-    | Archived
 
-  let renderTab = (~text: string, ~isActive: bool, ~onClick) => {
-    let active = "bg-gray-20 text-gray-80 rounded py-1"
-    <div
-      key=text
-      onClick
-      className={(
-        isActive ? active : "hover:cursor-pointer bg-white hover:text-gray-80"
-      ) ++ "  px-4 inline-block"}>
-      {React.string(text)}
-    </div>
-  }
+type category =
+  | /** Actually only unarchived */ All
```

Comment:
maybe it should then be called `Current` instead?

Reply:
I left it because it was used as a display label directly

Reply:
Well, the display label is also not accurate IMO. Not necessary to change it now though.